api/v1alpha4/ibmpowervs_conversion.go

```diff
@@ -164,3 +167,8 @@ func Convert_v1alpha4_IBMPowerVSMachineSpec_To_v1beta1_IBMPowerVSMachineSpec(in
 func Convert_v1beta1_IBMPowerVSMachineTemplateStatus_To_v1alpha4_IBMPowerVSMachineTemplateStatus(in *infrav1beta1.IBMPowerVSMachineTemplateStatus, out *IBMPowerVSMachineTemplateStatus, s apiconversion.Scope) error {
 	return autoConvert_v1beta1_IBMPowerVSMachineTemplateStatus_To_v1alpha4_IBMPowerVSMachineTemplateStatus(in, out, s)
 }
+
```

Comment:
I am not sure how correct these changes are!

Reply:
conversion changes are looking good. @Karthik-K-N